collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2343,53 +2333,50 @@ public void denied() {
                         }
                     });
                 } else {
-                    Intent reqIntent = getIntent();
-                    boolean showFirst = reqIntent.getBooleanExtra("start", false);
```

Comment:
In https://github.com/getodk/collect/pull/4286/commits/5b6ae1a3f2106b8395e3007964e65c3111b8a7e0 I just removed this check because we nowhere use `putBooleanExtra("start", false);` so this is always false. Please correct me if i made a mistake.

Reply:
I can't find anything on it and it doesn't look like it's documented (looking [here](https://docs.getodk.org/launch-collect-from-app/)) but I'm still a little worried it could be relied on somewhere out there? 

We can add a note to QA about this to see if they have any test cases for it. It doesn't look like the change would "break" anything if someone was using it - the user would just end up on the hierarchy.

Reply:
I can't find anything on it and it doesn't look like it's documented (looking [here](https://docs.getodk.org/launch-collect-from-app/)) but I'm still a little worried it could be relied on somewhere out there? 

We can add a note to QA about this to see if they have any test cases for it. It doesn't look like the change would "break" anything if someone was using it - the user would just end up on the hierarchy.

Reply:
I removed this change and I'll file a separate issue to discuss it.